pyupgrade/_plugins/unittest_aliases.py

```diff
@@ -12,7 +12,6 @@
 from pyupgrade._data import TokenFunc
 from pyupgrade._token_helpers import replace_name
 
-
```

Comment:
oops, one little thing -- or I can fix it -- up to you

Reply:
resolved